src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.ts

```diff
@@ -39,13 +41,22 @@ export class DotEditLayoutComponent implements OnInit {
                 'editpage.layout.toolbar.action.save',
                 'editpage.layout.toolbar.action.cancel',
                 'editpage.layout.toolbar.template.name',
-                'editpage.layout.toolbar.save.template'
+                'editpage.layout.toolbar.save.template',
+                'editpage.layout.dialog.edit.page',
+                'editpage.layout.dialog.edit.template',
+                'editpage.layout.dialog.info',
+                'editpage.layout.dialog.header'
             ])
             .subscribe();
 
         this.route.data.pluck('pageView').subscribe((pageView: DotPageView) => {
             this.pageView = pageView;
-            this.initForm(pageView);
+
+            if (!this.isLayout()) {
+                this.showTemplateLayoutSelectionDialog = true;
```

Comment:
I think with this new name and because we only do this here and once we don't need a method for this, I think it's pretty clear what this 3 lines does.